### DISCUSSION TITLE
onboarding config for github.com hosted app

### DISCUSSION THREAD

**Discussion by qmg-drettie** [renovatebot/renovate#18849] [Request Help] [answered]
I'd like to allow the github hosted app access to all our repos so that newly created repos receive an onboarding PR however, I don't want renovate to use the `extends:config:base`. I instead would like to extend from a custom config we have stored in a private repo

In all of our renovate enabled repos the config looks like this - 

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["github>qmetric/renovate-config"]
}
```

https://docs.renovatebot.com/self-hosted-configuration/#onboardingconfig
The `onboardingconfig` option seems to only be supported on self hosted.

Is something like the above possible on github app hosted renovate ?

**Comment by viceice** [comment, marked as answer]
you can't use private presets inside public repos.

otherwise you can use special shared org presets as onboarding config.

- `<org>/renovate-config/default.json`
- `<org>/.github/renovate-config.json`
